packages/platform-deploy/platform/interbit.manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "peers": [
-    "ib-dev----master.herokuapp.com",
-    "ib-dev-web-auth.herokuapp.com"
+    "ib-dev----master.herokuapp.com:443",
+    "ib-dev-web-auth.herokuapp.com:443"
```

Comment:
Why do we specify a port here?

Reply:
The heroku peers are using the default wss protocol port, 443. This will ensure the peers can connect to the wss socket on the other nodes correctly.